lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -149,17 +152,19 @@ function runLighthouse(url, configPath, isDebug) {
 function matchesExpectation(actual, expected) {
   if (typeof actual === 'number' && NUMERICAL_EXPECTATION_REGEXP.test(expected)) {
     const parts = expected.match(NUMERICAL_EXPECTATION_REGEXP);
-    const operator = parts[1];
-    const number = parseFloat(parts[2]);
+    const operator = parts[2];
+    const numbers = [parts[1], parts[3]].filter(p => typeof p !== 'undefined').map(parseFloat);
```

Comment:
I think it'd make more sense if you nixed the filter and made it explicit

```js
const [prefixNumber, operator, postfixNumber] = parts
// optional named descriptor, though IMO probably unnecessary and might just add cognitive overload
const targetNumber = typeof prefixNumber === 'undefined' ? postfixNumber : prefixNumber

Reply:
done. relying on implicit string -> float conversion. doesn't bother me here.

Reply:
I forgot about the `parseFloat` bit for a sec 😆 

maybe my "optional named descriptor" is more important to me than I realized... haha

Reply:
ha, it would probably have been simpler to just move off the switch statement :P